hw_1/task_2.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. Сколько журавликов сделал 
+# каждый ребенок, если известно, что Петя и Сережа сделали одинаковое 
+# количество журавликов, а Катя сделала в два раза больше журавликов, 
+# чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+
+S = int(input())
+k = int(S / 3 * 2)
+x = int(k / 4)
```

Comment:
Можно проще. К тому же не помешало бы проверить S на кратность 6
```suggestion
k = S // 3 * 2
x = k // 4
```